org.nickvision.money.json

```diff
@@ -0,0 +1,67 @@
+{
+    "app-id" : "org.nickvision.money",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "43",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "org.nickvision.money",
+    "finish-args":[
+        "--socket=fallback-x11",
+        "--socket=wayland",
+        "--device=dri",
+        "--share=ipc",
+        "--filesystem=host"
```

Comment:
why do you need host access?

Reply:
Please look at the note in PR description